tests/anndata/test_anndata_ext.py

```diff
@@ -393,17 +399,19 @@ def test_detect_mixed_binary_columns():
         {"Col1": list(range(4)), "Col2": ["str" + str(i) for i in range(4)], "Col3": [1.0, 0.0, np.nan, 1.0]}
     )
     adata = ep.ad.df_to_anndata(df)
+    ep.ad.infer_feature_types(adata, output=None)
+
     assert_frame_equal(
         adata.var,
         DataFrame(
-            {EHRAPY_TYPE_KEY: [NUMERIC_TAG, NON_NUMERIC_TAG, NUMERIC_TAG]},
+            {FEATURE_TYPE_KEY: [NUMERIC_TAG, CATEGORICAL_TAG, CATEGORICAL_TAG]},
             index=["Col1", "Col2", "Col3"],
         ),
     )
 
 
 @pytest.fixture
-def adata_numeric():
+def adata_numeric():  # TODO: Delete: I don't think this is used
```

Comment:
I couldn't find any tests using this fixture while going through the tests. Are the tests missing, or should I simply remove the `adata_numeric` method?

Reply:
You may remove it if it's not being used :)

I'll overhaul all fixtures in a couple of weeks so that we have less and more useful ones that are centralized in conftest.py